multidim-interop/impl/dotnet/v1.0/.gitignore

```diff
@@ -0,0 +1,5 @@
+bin/
+obj/
+.vs/
+.vscode/
```

Comment:
Please no editor specific configurations in the project `.gitignore`. These should go into your home `~/.gitignore`.